docs/builtin_workflows.rst

```diff
@@ -510,6 +510,178 @@ the model-fitting workflows and which sampling schemes work with them.
 
 \* Not recommended
 
+.. _connectivity_matrices:
+
+*********************
+Connectivity matrices
+*********************
```

Comment:
I just realized where you put this. Should this go in `outputs`?

Reply:
It's kind of outside of individual workflows (they all make these matfiles), but is slightly different for different workflows (the variables are named differently for dsi studio and mrtrix3)